kubernetes/crypto_detector/crypto_detection.yaml

```diff
@@ -17,6 +17,7 @@ spec:
         image: kubernetes:31320/crypto-detector-system
         ports:
         - containerPort: 12000
+        args: ["-b", "kafka-palantir:9092", "-c", "cds-input", "-p", "cds-result"]
```

Comment:
Related to my previous comment on the Dockerfile. Since you rely on `ENTRYPOINT`, the arguments that you list here will be appended to the code specified within the `ENTRYPOINT`. Therefore, it seems as if you are actually overriding the default values by duplicating the input arguments (last argument values take precedence).